src/Registration/cReg/NiftiImageData.cpp

```diff
@@ -536,7 +536,7 @@ float NiftiImageData<dataType>::get_norm(const NiftiImageData<dataType>& other)
         if (!std::isnan(this->operator()(i)+other(i)))
             result += double(pow( this->operator()(i) - other(i), 2));
```

Comment:
Possibly unrelated to this PR, but can `dataType` be `std::complex<>`? Maybe needs `std::abs()`^2 instead.